app/views/about/privacy.en.html.erb

```diff
@@ -5,8 +5,30 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">Privacy notice</h1>
-        <p class="govuk-body">This privacy notice explains what the Ministry of Justice (MOJ) means when we talk about personal data, why we ask for this information about you and what we do with it when you use the ‘<%= service_name -%>’ online service.</p>
-        <p class="govuk-body">TODO: Add in appropriate content</p>
+        <p class="govuk-body">This service is completely anonymous to use. Users are not asked for any personal information that could be used to identify them (such as their name, address or prisoner number).</p>
+
+        <h2 class="govuk-heading-m">Making a complaint</h2>
+
+        <p class="govuk-body">When we ask you for information, we will keep to the law. If you think that your information has not been handled correctly, you can contact the Information Commissioner for independent advice about data protection on the address below:</p>
+
+        <div class="govuk-inset-text">
+          <address class="adr">
+            <div class="extended-address">Information Commissioner's Office,</div>
+            <span class="extended-address">Wycliffe House,</span>
+            <span class="street-address">Water Lane,</span>
+            <div class="locality">Wilmslow,</div>
+            <span class="region">Cheshire,</span>
+            <span class="postal-code">SK9 5AF</span>
+          </address>
+
+          <div class="util_mt-medium">
```

Comment:
Ok this is almost ready.
Minor details:

* The class `util_mt-medium` doesn't exist in the new design system. We can replace it for, more or less the equivalent, `govuk-!-margin-top-3`
* Let's add the `govuk-link` class to the ICO link so it is the right shade of blue.
* Optional: we might need to override the `address` default italic styling, as I seem to remember italic doesn't play well in accessibility tests. This probably will need to be confirmed with Matt and Simon, for now we can leave it as italic (browsers' default) but can you confirm with them please.